app/src/main/java/com/orange/ouds/app/ui/components/badge/BadgeDemoScreen.kt

```diff
@@ -80,12 +85,12 @@ private fun BadgeDemoBottomSheetContent(state: BadgeDemoState) {
             label = stringResource(id = R.string.app_components_common_status_label),
             items = statuses.map { status ->
                 CustomizationDropdownMenuItem(
-                    label = status.name.toSentenceCase(),
```

Comment:
We should keep the call to `toSentenceCase` just in case a new status with two words is added.